qtfred/src/mission/dialogs/CustomWingNamesDialogModel.cpp

```diff
@@ -0,0 +1,119 @@
+#include "CustomWingNamesDialogModel.h"
+
+#include "ship/ship.h"
+
+namespace fso {
+namespace fred {
+namespace dialogs {
+
+CustomWingNamesDialogModel::CustomWingNamesDialogModel(QObject * parent, EditorViewport * viewport) :
+	AbstractDialogModel(parent, viewport) {
+	initializeData();
+}
+
+bool CustomWingNamesDialogModel::apply() {
+	int i;
+	if (strcmp(_m_starting[0].c_str(), _m_tvt[0].c_str()))
+	{
+		//MessageBox("The first starting wing and the first team-versus-team wing must have the same wing name.");
```

Comment:
You should be able to display this message with the dialog provider of `_editor`.